src/engine_openssl.c

```diff
@@ -891,21 +894,30 @@ static int generate_csr(tls_private_key key, char **pem, size_t *pemlen, ...) {
         X509_NAME_add_entry_by_txt(subj, id, MBSTRING_ASC, val, -1, -1, 0);
     }
 
+#define ssl_check(OP) do{ \
+op = #OP;                          \
+if((OP) == 0) {             \
+ret = ERR_get_error();\
+goto on_error;\
```

Comment:
i appreciate that not a single line has the continuation character at the same location.